bigbluebutton-html5/imports/ui/components/whiteboard/annotations/text/component.jsx

```diff
@@ -138,9 +138,11 @@ export default class TextDrawComponent extends Component {
 
   renderViewerTextShape(results) {
     const styles = TextDrawComponent.getViewerStyles(results);
+    const { isChrome, isEdge } = this.props.browserType;
 
     return (
       <g>
+        { isChrome || isEdge ? null :
```

Comment:
This will prevent textShapes from showing in Chrome and Edge? :thinking:

Reply:
I tested, it doesn't